pages/linux/grafana-cli.md

```diff
@@ -0,0 +1,16 @@
+# grafana-cli
+
+> A small executable that is bundled with Grafana serve.
+> More information: <https://grafana.com>.
+
+- Install a grafana plugin:
```

Comment:
```suggestion
- Install/update/remove specific plugins:
```

This page already describes `grafana-cli`, it's not needed to mention it explicitly.